src/assets/base.css

```
@@ -0,0 +1,124 @@
+*,::before,::after{
+ margin: 0;
+ box-sizing: border-box;
+}
+
+body {
+ width: 100vw;
+ height: 100vh;
+ overflow: hidden;
```

This should help with scrolling on mobile!

```suggestion
 min-height: 100vh;
 overflow-x: hidden;
```